File: src/main/java/jp/aoichaan0513/A_TosoGame_Live/API/Maps/PlotInfo.java
package jp.aoichaan0513.A_TosoGame_Live.API.Maps;

import org.bukkit.Location;
import org.bukkit.map.MapCursor;

public class PlotInfo {

    public Location loc;
    public boolean setup;
    public MapCursor cursor;
    public MapCursor.Type type;

    public PlotInfo(Location loc, MapCursor.Type color) {
        this.loc = loc;
        setup = false;
        type = color;
    }

    public boolean isSetup() {
        return setup;
    }

    public void setSetup(boolean setup) {
        this.setup = setup;
    }

}

File: src/main/kotlin/jp/aoichaan0513/A_TosoGame_Live/API/Map/PlotInfo.kt
package jp.aoichaan0513.A_TosoGame_Live.API.Map

import org.bukkit.Location
import org.bukkit.map.MapCursor

class PlotInfo(var loc: Location, var type: MapCursor.Type) {
    var isSetup = false
    var cursor: MapCursor? = null

}